### ISSUE TITLE
[Bug]: Installation fails class not foud

### ISSUE THREAD

**Issue by OttschoChris** [HEPTACOM/HeptacomShopwarePlatformAdminOpenAuth#25]
### Plugin Version

6.0.3

### PHP Version

8.2.19

### Shopware Version

6.5.7.3

### Installation method

Community-Store

### Identity provider

None

### What happened?

Installation fails with the error in the log.
Tried deletion and reinstall.

### Relevant log output

```shell
“The class “Heptacom\AdminOpenAuth\KskHeptacomAdminOpenAuth” is not found. Probably an class loader error. Check your plugin composer.json”
```

**Comment by ottschoNaud** [comment]
We installed it via administration UI, and also tried CLI bin/console way. Same result for both

**Comment by JoshuaBehrens** [comment]
So you install it via ZIP file, right? I admit we do not test this. We only do composer managed projects ^^' If you confirm this, we will have a look. Otherwise I suggest to install via composer if this is suitable for you

**Comment by ottschoNaud** [comment]
Yes we used the zip file or better download from store functionality. Installation via composer worked!
